src/frontend/js/widgets/Dashboard/components/DashboardSidebar/index.tsx

```diff
@@ -61,11 +55,12 @@ export const DashboardSidebar = () => {
             <DashboardAvatar user={user!} />
           </div>
           <h3>
-            <FormattedMessage {...messages.header} values={{ name: user?.username }} />
+            <FormattedMessage {...header} values={{ name: user?.username }} />
           </h3>
           <p>
-            <FormattedMessage {...messages.subHeader} />
+            <FormattedMessage {...subHeader} />
           </p>
+          {settingsUrl && <SettingsLink to={settingsUrl} />}
```

Comment:
We would maybe make this more abstract by just providing `headerChildren` props where we can put any ReactNode we want? WDYT?

Reply:
I don't know.
This button is part of the sidebar template, it would be nice to use children if the placement is used for different elements.
Here it will always be either a settings link, either nothing. I feel like the `settingsUrl` prop is the best.

Reply:
This button isn't part of the PR anymore